http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -749,25 +729,45 @@ object Boot extends IOApp {
               nonLeoMessageSubscriber
             )
           }
+          //This where Azure Dependencies are created when hosting Leonardo on Azure.
         case None =>
-          val publisher = new LeoPublisher(publisherQueue, new AzurePublisher[IO] {}) // TODO: Jesus will fix this
+          val pubConfig = AzureServiceBusPublisherConfig(
```

Comment:
don't feel strongly, but we typically have this set up in config file to keep the `Boot` file a bit shorter...and we can reuse config value within config file...